sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3125,6 +3125,10 @@ SIREPO.app.directive('simStatusPanel', function(appState) {
                   '<div data-sbatch-cores-and-hours="simState"></div>',
                 '</div>',
               '</div>',
+            '<div data-ng-if="cancelledDueToTimeout()" class="alert alert-warning" role="alert">',
+                '<h4 class="alert-heading">Cancelled: Maximum runtime exceeded</h4>',
+                '<p>Your simulation ran longer than the maximum runtime. To increase your maximum runtime please upgrade to <a href="https://radiasoft.net/sirepo" target="_blank">Sirepo Premium</a>.</p>',
```

Comment:
If they are already Premium, we want them to upgrade to Enterprise. Therefore, authState needs isPremiumUser.

Reply:
authState has roles in it so I'm now checking those.

https://github.com/radiasoft/sirepo/pull/2373/files#diff-138a2b807a59f609d46eb468d799eaa1R326